### PR TITLE
update publish message with studio.mapbox.com link

### PR DESCRIPTION
**Pull request by mapsam**
Adds https://studio.mapbox.com/tilesets/* to the publish command output

cc @millzpaugh

### CODE REVIEW

**Comment by mapsam** [comment]
### pre-commit updates

Updated the pre-commit configuration with `pre-commit autoupgrade` as per the warning in the console. This led me to find 
https://github.com/pre-commit/pre-commit-hooks/issues/344 and so I updated the repo to point to gitlab.

File: setup.py [hunk 1]
```diff
@@ -40,7 +40,14 @@ def read(fname):
...
     zip_safe=False,
     extras_require={
-        "test": ["pytest==4.6.11", "pytest-cov", "pre-commit", "black", "pep8"]
+        "test": [
+            "pytest==4.6.11",
+            "pytest-cov",
+            "pre-commit",
+            "black==20.8b1",
+            "pep8",
+            "toml==0.10.2",
```
Comment (mapsam) [inline setup.py:49]: black 20.8b1 would not install without explicitly updating toml to >=0.10.1

File: mapbox_tilesets/scripts/cli.py [hunk 2]
```diff
@@ -119,10 +119,19 @@ def publish(tileset, token=None, indent=None):
...
-        click.echo(json.dumps(r.json(), indent=indent))
+        response_msg = r.json()
+        click.echo(json.dumps(response_msg, indent=indent))
+
+        studio_url = click.style(
+            f"https://studio.mapbox.com/tilesets/{tileset}", bold=True
+        )
+        job_id = response_msg["jobId"]
+        job_cmd = click.style(f"tilesets job {tileset} {job_id}", bold=True)
+        message = f"\n✔ Successfully published. Visit {studio_url} or run {job_cmd} to view the status of your tileset."
```
Comment (ericfischer) [inline mapbox_tilesets/scripts/cli.py:130]: Tangential to the substance of the change, but should we really say "successfully published" here when we don't know yet whether the actual publication will succeed or fail? Could it be "publication started" or something like that instead?

**Review by ericfischer — APPROVED** [review]
Looks good, aside from wording concern

File: mapbox_tilesets/scripts/cli.py (hunk 2 above)
Comment (mapsam) [reply mapbox_tilesets/scripts/cli.py:130]: Good call out @ericfischer - I want to refrain from adding more verbiage to an already verb-heavy endpoint. My perspective says that the request to publish this tileset was successful. Plus the sentence suggesting how to view the "status" of your tileset reads as if it's not guaranteed the publish will result in a complete tileset. 

I do like the idea of saying something like:

```
✔ Tileset job recieved. <...> 
```

What do you think?

Comment (ericfischer) [reply mapbox_tilesets/scripts/cli.py]: Sounds great! Thanks for changing it.

**Comment by mapsam** [comment]
Thanks @ericfischer! I'll publish this as 1.7.0 and merge to master.

**Pull request merged**